{
  "test_status": "failed",
  "exception": {
    "class": "AssertionError",
    "message": "assert False"
  }
}
